locales/en.json

```diff
@@ -291,7 +291,8 @@
       },
       "bleError": {
         "title": "Failed to transfer",
-        "message": "Something went wrong while transferring {{vcLabel}}. Please try again."
+        "message": "Something went wrong while transferring {{vcLabel}}. Please try again.",
+        "hint": "Error: {{code}}"
```

Comment:
We need this new field to be added for other languages right? Otherwise it may show up as `bleError.hint` in the UI for other languages. Can you confirm and fix this?

Reply:
Added Translations.